src/app/components/passphrase-form/passphrase-form.component.html

```diff
@@ -23,6 +24,27 @@
       </div>
     </div>
   </div>
+
+
+
+  <div class="form-group row">
+    <label for="walletPassphrase" class="col-sm-3 col-form-label"
+      >Wallet Enckey</label
+    >
+    <div class="col-sm-9">
+      <input
+        id="walletEnckey"
+        type="text"
+        name="walletEnckey"
+        class="form-control"        
+        placeholder="Wallet Enckey"
+        [(ngModel)]="walletEnckey"
+      />      
+    </div>
```

Comment:
Inputting "encKey" isn't very "human" interface

Reply:
i'll store enckey to client

Reply:
ok

Reply:
not store, but one can use that "auth" command to derive it

Reply:
yes, i'll use auth command to get it.